share/functions/__fish_git_prompt.fish

```diff
@@ -367,7 +367,7 @@ function __fish_git_prompt --description "Prompt function for Git"
         else
             if test -n "$__fish_git_prompt_showdirtystate"
                 set -l config (command git config --bool bash.showDirtyState)
-                if test "$config" != "false"
+                if test "$config" != false
```

Comment:
It was for consistency. Just 10 lines later there's a same test, without quotes.
I can remove it if it's such an issue...